components/network/asterisk/asterisk.p5m

```diff
@@ -0,0 +1,1002 @@
+file path=var/lib/asterisk/sounds/en/confbridge-locked.gsm
+file path=var/lib/asterisk/sounds/en/vm-undelete.gsm
+file path=var/lib/asterisk/sounds/en/vm-pls-try-again.gsm
+file path=var/lib/asterisk/sounds/en/conf-unmuted.gsm
+file path=var/lib/asterisk/sounds/en/vm-Cust4.gsm
+file path=var/lib/asterisk/sounds/en/vm-invalidpassword.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-rest-talk-vol-out.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-pin-bad.gsm
+file path=var/lib/asterisk/sounds/en/vm-msgsaved.gsm
+file path=var/lib/asterisk/sounds/en/conf-extended.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-dec-talk-vol-in.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-inc-talk-vol-out.gsm
+file path=var/lib/asterisk/sounds/en/vm-saveoper.gsm
+file path=var/lib/asterisk/sounds/en/tt-monkeysintro.gsm
+file path=var/lib/asterisk/sounds/en/spy-nbs.gsm
+file path=var/lib/asterisk/sounds/en/priv-callpending.gsm
+file path=var/lib/asterisk/sounds/en/beeperr.gsm
+file path=var/lib/asterisk/sounds/en/conf-hasjoin.gsm
+file path=var/lib/asterisk/sounds/en/dictate/record_help.gsm
+file path=var/lib/asterisk/sounds/en/dictate/enter_filename.gsm
+file path=var/lib/asterisk/sounds/en/dictate/truncating_audio.gsm
+file path=var/lib/asterisk/sounds/en/dictate/play_help.gsm
+file path=var/lib/asterisk/sounds/en/dictate/record_mode.gsm
+file path=var/lib/asterisk/sounds/en/dictate/playback.gsm
+file path=var/lib/asterisk/sounds/en/dictate/paused.gsm
+file path=var/lib/asterisk/sounds/en/dictate/record.gsm
+file path=var/lib/asterisk/sounds/en/dictate/both_help.gsm
+file path=var/lib/asterisk/sounds/en/dictate/pause.gsm
+file path=var/lib/asterisk/sounds/en/dictate/playback_mode.gsm
+file path=var/lib/asterisk/sounds/en/dictate/forhelp.gsm
+file path=var/lib/asterisk/sounds/en/spy-mgcp.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-unlocked.gsm
+file path=var/lib/asterisk/sounds/en/conf-lockednow.gsm
+file path=var/lib/asterisk/sounds/en/priv-recordintro.gsm
+file path=var/lib/asterisk/sounds/en/confbridge-dec-list-vol-out.gsm
+file path=var/lib/asterisk/sounds/en/agent-alreadyon.gsm
+file path=var/lib/asterisk/sounds/en/vm-tocancel.gsm
+file path=var/lib/asterisk/sounds/en/vm-tempgreeting2.gsm
+file path=var/lib/asterisk/sounds/en/agent-user.gsm
+file path=var/lib/asterisk/sounds/en/vm-tocallback.gsm
+file path=var/lib/asterisk/sounds/en/demo-echotest.gsm
+file path=var/lib/asterisk/sounds/en/vm-sorry.gsm
+file path=var/spool/asterisk/voicemail/default/1234/en/busy.gsm
+file path=var/spool/asterisk/voicemail/default/1234/en/unavail.gsm
+file path=usr/share/man/man8/autosupport.8
+file path=usr/share/man/man8/safe_asterisk.8
+file path=usr/share/man/man8/astgenkey.8
+file path=usr/share/man/man8/asterisk.8
+file path=usr/sbin/astgenkey
+file path=usr/sbin/autosupport
+file path=usr/sbin/safe_asterisk
+file path=usr/sbin/astcanary
+file path=usr/sbin/asterisk
+<transform file path=etc/.* -> default preserve true>
```

Comment:
please, move it to the first lines of p5m file ( near license, so that it could be visibe).